Dockerfile

```diff
@@ -45,18 +64,6 @@ RUN go build
 WORKDIR /go/mx-chain-proxy-go-mainnet/cmd/proxy
 RUN go build
 
```

Comment:
not related to this, maybe in a future PR, we have also for proxy `appVersion` and `commitID` based on build data, so it can be build like this `-ldflags="-X main.appVersion=$(git --git-dir /workspace/mx-chain-devnet-config/.git describe --tags --long --dirty) -X main.commitID=$(git --git-dir /workspace/mx-chain-devnet-config/.git rev-parse HEAD)"`

Reply:
Partially fixed - `appVersion`, but without `commitID` - as for the Node.